site/app/entities/course/CourseMaterial.php

```diff
@@ -81,8 +108,11 @@ public function __construct(int $type, string $path, \DateTime $release_date, bo
         $this->sections = new ArrayCollection();
         $this->url = $url;
         $this->title = $title;
+        $this->uploaded_by = $uploaded_by;
+        $this->upload_date = $upload_date;
+        $this->last_edit_by = $last_edit_by;
+        $this->last_edit_date = $last_edit_date;
     }
-
```

Comment:
This whitespace change should be reverted.

Reply:
Sure.